dox/CMakeLists.txt

```diff
@@ -9,20 +9,14 @@
 cmake_minimum_required (VERSION 3.0.0)
 project (seqan_dox CXX)
 
-# require python 2.7, not python3
-set(PythonInterp_FIND_VERSION 2.7)
-set(PythonInterp_FIND_VERSION_MAJOR 2)
-set(PythonInterp_FIND_VERSION_MINOR 7)
-set(PythonInterp_FIND_VERSION_COUNT 2)
-
-find_package (PythonInterp)
-if (NOT PYTHONINTERP_FOUND OR NOT(${PYTHON_VERSION_MAJOR} MATCHES "2"))
-    message (STATUS "  You need Python 2.x for building dox. (skip the tests)")
+find_package (Python3)
```

Comment:
I think this is not necessary.

Reply:
See my general comment. IMHO I think being explicit is what we want here.

Reply:
`set(PythonInterp_FIND_VERSION 3)` is as explicit as your method, as it only accepts versions of at least python 3. And we can keep our minimum version of cmake. Saying that, since cmake 3.12 this module is deprecated in favour of `find_package(Python)` :frowning:

Reply:
Well, why are we sticking to an old cmake version if we can get with pip a much more recent one which would work for all our supported platforms?